### PR TITLE
Fix quicklink icon alignment

### PR DESCRIPTION
**Pull request by lookacat**
## Description
We've fixed the alignment of the quicklink icons.
(There is no better solution but to hard set pixel values because the icons itself have the exact same size and padding, its just because of the form of the icons that it looks like the padding is wrong)
For more details see https://github.com/owncloud/web/issues/8606

## Related Issue

- Fixes https://github.com/owncloud/web/issues/8606

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue [hunk 1]
```diff
@@ -525,4 +525,9 @@ export default defineComponent({
     justify-content: flex-start;
   }
 }
+
+.oc-files-public-link-expires {
+  margin-top: 1px; // to align with the other elements
+  margin-left: 4px; // to align with the other elements
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue:531]: Please use `var(--oc-space-xsmall)`

Comment (lookacat) [reply packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue]: yes

**Review by kulmann — CHANGES_REQUESTED** [review]
Please also adjust for the icon in the sharing panel, see screenshot

Screenshot 2023-04-17 at 13 09 11

**Comment by ChrisEdS** [comment]
> Please also adjust for the icon in the sharing panel, see screenshot
> 
> Screenshot 2023-04-17 at 13 09 11

I'm I wrong or are these two different icons for the same thing (expiration date)?

image

and 

image

**Comment by kulmann** [comment]
@ChrisEdS correct, good spot. Let’s only use one. 🙈 I prefer the one we have for links. You?

**Comment by ChrisEdS** [comment]
Yep, seems like the one without the "date picked" is used as "remove date" icon

image

**Comment by lookacat** [comment]
ok will look into it when i'm done with the other ticket

**Comment by lookacat** [comment]
done

File: packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue [hunk 2]
```diff
@@ -409,4 +409,7 @@ export default defineComponent({
     margin-left: var(--oc-space-medium);
   }
 }
+.files-collaborators-collaborator-expiration {
+  margin-top: 5px;
```
Comment (JanAckermann) [inline packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue:413]: 4px not possible?

File: packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue [hunk 3]
```diff
@@ -525,4 +525,9 @@ export default defineComponent({
     justify-content: flex-start;
   }
 }
+
+.oc-files-public-link-expires {
+  margin-top: 1px; // to align with the other elements
+  margin-left: var(--oc-space-xsmall); // to align with the other elements
```
Comment (JanAckermann) [inline packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue:531]: please in class

**Review by JanAckermann — CHANGES_REQUESTED** [review]

File: packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue (hunk 2 above)
Comment (lookacat) [reply packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue:413]: looks wrong

Comment (lookacat) [reply packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue:413]: 1px off

**Review by JanAckermann — APPROVED** [review]

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**